libs/perun/components/src/lib/attributes-list/attributes-list.component.html

```diff
@@ -43,7 +43,14 @@
       <ng-container matColumnDef="displayName">
         <th *matHeaderCellDef mat-header-cell
             mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.ATTRIBUTES_LIST.NAME' | translate}}</th>
-        <td mat-cell *matCellDef="let attribute">{{attribute.displayName}}</td>
+        <td mat-cell
+            *matCellDef="let attribute">
+          <span
+            matTooltipClass='urn-tooltip'
+            matTooltip='{{getAttributeFullName(attribute)}}'>
+            {{attribute.displayName}}
+          </span>
```

Comment:
If you have some multi line `displayName`, then the tooltip hides between every single line on hovering. Maybe a `div` instead of `span` could solve it? But it's only a small suggestion, which is not necesary.

Reply:
I'm not sure, what do you mean. Could you please provide me a screenshot?

Reply:
I have replaced the `span` element with `div` and it fixed the problem.